frontend/src/APIClients/mutations/AuthMutations.ts

```diff
@@ -1,7 +1,7 @@
 import { gql } from "@apollo/client";
 
-export const REGISTER = gql`
```

Comment:
`register` resolver will set the cookies for the current user, but i don't think that would make sense for adding admin since the user != new admin and we only want to create user + send email, which is what `createUser` does

also using the same resolver might not work since the function param `CreateTeacherDTO` would have extra fields unnecessary for `CreateAdminDTO`

but i agree that having them in separate resolvers is a little confusing - would it help to have a `registerTeacher` and `registerAdmin` resolver?

Reply:
Ohhhh, I see. For your last point, I think the separate resolvers makes sense to me! To your discretion though, if you think it's fine as is, nbd, I'm excited for your PR to get merged :D